src/js/player.js

```diff
@@ -312,6 +312,9 @@ class Player extends Component {
     // Run base component initializing with new options
     super(null, options, ready);
 
+    // was a source set intially? used to determine if we should fire a sourcechange
+    this.initialSourceSet_ = tag.src ? true : false;
```

Comment:
why not `!!tag.src`?

Reply:
that would work too

Reply:
seems like it doesn't work as intended as tag.src returns weird things on IE